src/nvim/ui.c

```diff
@@ -598,6 +615,15 @@ Array ui_array(void)
     PUT(info, "height", INTEGER_OBJ(ui->height));
     PUT(info, "rgb", BOOLEAN_OBJ(ui->rgb));
     PUT(info, "override", BOOLEAN_OBJ(ui->override));
+
+    // TUI fields.
+    PUT(info, "term_name", STRING_OBJ(cstr_to_string(ui->term_name)));
+    PUT(info, "term_background", STRING_OBJ(cstr_to_string(ui->term_background)));
```

Comment:
Wondering if `term_background` and `stdin_fd` are too low-level to include in this UI description. Added them for completeness/consistency...

Reply:
`stdin_fd` should not be included. The fd will only be used early in startup and then closed.